airflow/contrib/example_dags/example_gcp_transfer.py

```diff
@@ -0,0 +1,285 @@
+
+* GCP_PROJECT_ID - Google Cloud Project to use for the Google Cloud Transfer Service.
+* GCP_SOURCE_BUCKET - Google Cloud Storage bucket from which files are copied.
+* GCP_TARGET_BUCKET - Google Cloud Storage bucket bucket to which files are copied
+
+"""
+import os
+from datetime import datetime, timedelta
+from copy import deepcopy
+
+from airflow import models
+from airflow.contrib.hooks.gcp_transfer_hook import GcpTransferOperationStatus
+from airflow.contrib.operators.gcp_transfer_operator import (
+    GcpTransferServiceJobCreateOperator,
+    GcpTransferServiceJobDeleteOperator,
+    GcpTransferServiceJobUpdateOperator,
+    GcpTransferServiceOperationsListOperator,
+    GcpTransferServiceOperationGetOperator,
+    GcpTransferServiceOperationPauseOperator,
+    GcpTransferServiceOperationResumeOperator,
+    GcpTransferServiceOperationCancelOperator)
+from airflow.contrib.sensors.gcp_transfer_sensor import GCPTransferServiceWaitForJobStatusSensor
+from airflow.utils.dates import days_ago
+
+# [START howto_operator_gct_common_variables]
+
+GCP_PROJECT_ID = os.environ.get('GCP_PROJECT_ID', 'example-project')
+GCP_DESCRIPTION = os.environ.get('GCP_DESCRIPTION', 'description')
+GCP_TRANSFER_TARGET_BUCKET = os.environ.get('GCP_TRANSFER_TARGET_BUCKET')
+WAIT_FOR_OPERATION_POKE_INTERVAL = os.environ.get('WAIT_FOR_OPERATION_POKE_INTERVAL', 5)
+
+GCP_TRANSFER_SOURCE_AWS_BUCKET = os.environ.get('GCP_TRANSFER_SOURCE_AWS_BUCKET')
+GCP_TRANSFER_FIRST_TARGET_BUCKET = os.environ.get('GCP_TRANSFER_FIRST_TARGET_BUCKET',
+                                                  'gcp-transfer-first-target')
+GCP_TRANSFER_SECOND_TARGET_BUCKET = os.environ.get('GCP_TRANSFER_SECOND_TARGET_BUCKET',
+                                                   'gcp-transfer-second-target')
+# [END howto_operator_gct_common_variables]
+
+now = datetime.utcnow()
+job_time = now + timedelta(minutes=2)
+
+# [START howto_operator_gct_create_job_body_aws]
+first_transfer_create_body = {
+    "description": GCP_DESCRIPTION,
+    "status": "ENABLED",
+    "projectId": GCP_PROJECT_ID,
+    "schedule": {
+        "scheduleStartDate": {
+            "day": 1,
+            "month": 1,
+            "year": 2015
+        },
+        "scheduleEndDate": {
+            "day": 1,
+            "month": 1,
+            "year": 2020
+        },
+        "startTimeOfDay": {
+            "hours": job_time.hour,
+            "minutes": job_time.minute
+        }
+    },
+}
+# [END howto_operator_gct_create_job_body_aws]
+
+# [START howto_operator_gct_update_job_body]
+update_body = {
+    "projectId": GCP_PROJECT_ID,
+    "transferJob": {
+        "description": "%s_updated" % GCP_DESCRIPTION
+    },
+    "updateTransferJobFieldMask": "description"
+}
+# [END howto_operator_gct_update_job_body]
+
+list_filter_dict = {
+    "projectId": GCP_PROJECT_ID,
+    "jobNames": []
+}
+
+second_transfer_create_body = deepcopy(first_transfer_create_body)
```

Comment:
Lepiej nie bawić się deepcopy, bo to będzie w dokumentacji, a dokumentacji musi to być czytelne i łatwe do skopiowania.